_css/styles.css

```diff
@@ -1 +1,30 @@
 #map{width:100%;height:400px;margin-bottom:2em}#content{padding-bottom:40px;text-align:center}#content.home{padding-top:40px}.navbar-brand a{color:#fff}.container-fluid{padding:0}.form-control-file{width:auto;margin:0 auto}
+
+.upload-text{
+    background: #a8d6e0;
+    border-radius: 5px;
+    max-width: 350px;
+    text-align: center;
+    margin: 30px auto;
+}
+
+.upload-text p {
+    color: #fff;
+    font-size: 40px;
+    margin: 0;
+    padding: 50px 0;
+}
+.upload-text i {
+	opacity: 1;
+	font-size: 30px;
+	display: block;
+}
+
+.upload-text .up {
+	font-weight: bold;
+}	
+
+.form-group label{
+	font-size: 40px;
+	font-weight: bold;
+}
```

Comment:
Please move your css to my sass file and compile to the css file. You can use an application like Scout Sass compiler